### PR TITLE
Better debugging log for investigating pod unknown status.

### PR DESCRIPTION
**Pull request by dchen1107**

### CODE REVIEW

File: pkg/master/pod_cache.go [hunk 1]
```diff
@@ -172,7 +172,7 @@ func (p *PodCache) computePodStatus(pod *api.Pod) (api.PodStatus, error) {
 
 	// Assigned to non-existing node.
 	if err != nil || len(nodeStatus.Conditions) == 0 {
-		glog.V(5).Infof("node doesn't exist: %v %v, setting pod status to unknown", err, nodeStatus)
+		glog.V(5).Infof("node doesn't exist: %v %v, setting pod %s status to unknown", err, nodeStatus, pod.Name)
```
Comment (saad-ali) [inline pkg/master/pod_cache.go]: Should use %q so that empty strings are obvious

**Comment by saad-ali** [comment]
LGTM mod nit

**Comment by dchen1107** [comment]
Addressed the comment, and PTAL?

**Comment by saad-ali** [comment]
LGTM, merging

**Pull request merged**